src/en/space-station-14/departments/security/proposals/rework-warden.md

```diff
@@ -0,0 +1,85 @@
+Warden's role is designed in a way that contradicts one of the most important rules of SS14 design. This is bad.
+
+P.S. Also keep in mind that Warden's gameplay loop should improve with GenPop.
+
+## _transformSystem.Unanchor(warden)
+
+Warden needs more interactions with other departments, which should encourage warden to sometimes leave their nest or at least give them more things to do while guarding security.
+Here are some problems that a warden can encounter:
+```mermaid
+flowchart LR;
+    id1["Prisoner is hurt"] --> id2(["Medical"])
+    id4["Secfab needs materials"] --> id5(["Cargo"])
+    id6["Prisoner needs food/drink"] -->|This does NOT happen| id7(["Service"])
+    id8["New prisoner needs sentencing"] --> id9{"Cell/perma"}
+    id10["Attack on security area"] --> id11(["Security"])
+    id12["Something sus on cameras"] --> id11
+    id13["Wanted records need an update"] --> id14{Records Computer}
+```
+Out of these problems, only 2 actually require interactions outside of security - healing prisoners and getting materials.
+I'll cover each of these interactions and how to improve them.
+
+## Prisoners must be granted adequate medical care
+Healing prisoners is often done by security officers, since they are mobile and can actually escort a prisoner to medical and back. Instead, I suggest something a few maps have: a common area available both to security and medical. I'll refer to it as "Infirmary Wing".
+
+![Example of infirmary wing on Cluster](https://i.imgur.com/PGeTpOy.png)
+
+The intended way the interaction with medical will go is like this:
+- Warden calls a doctor to the infirmary wing;
+- Warden escorts a hurt prisoner to infirmary;
+- Doctor arrives and heals the prisoner;
+- Warden escorts the prisoner back into the cell/perma.
+
+## Cargo interaction
+Interaction with cargo is mostly good already, but there is something to add here. A lot of crates in cargo contain security gear - it would be best to make it a little bit more difficult to order those by duplicating announcements to security channel when something security-related gets ordered. This can still be bypassed via antagonist items like jammers, but prevents things like salvagers ordering a crate of guns and breaking it open without any communication.
+While this isn't related specifically to warden, it is still a pretty nice improvement overall.
+
+## Food and drink for prisoners
+Prisoners have right to food and drink per space law, but this is difficult to provide as of now: in order to give prisoner something, you need to open the cell, which forces you to use a flash if the prisoner wants to escape.
+
+A solution to this is a special cell door with a container slot - you will be able to insert items into the door, and the prisoner can get them out of the door. If meal trays ever get added, maybe it'd make sense to map some near the cells. This will (hopefully) result in more situations where prisoners ask for a meal/drink and warden actually provides it.
```

Comment:
No foodboxes exist like this because there's supposed to be a risk of going in to feed prisoners (assuming no vendors or botany is in perma for prisoners to vend or grow their own food). This removes an important aspect of _risk_